Stellar-ledger.x

```diff
@@ -488,13 +488,27 @@ case 3:
 // This struct groups together changes on a per transaction basis
 // note however that fees and transaction application are done in separate
 // phases
-struct TransactionResultMeta
+struct TransactionResultMetaV0
 {
     TransactionResultPair result;
     LedgerEntryChanges feeProcessing;
     TransactionMeta txApplyProcessing;
 };
 
+// This struct groups together changes on a per transaction basis
+// note however that fees and transaction application are done in separate
+// phases
+struct TransactionResultMetaV1
+{
+    ExtensionPoint ext;
+
+    TransactionResultPair result;
+    LedgerEntryChanges feeProcessing;
+    TransactionMeta txApplyProcessing;
+
+    LedgerEntryChanges postTxApplyProcessing;
```

Comment:
nit: This should probably be called `postTxApplyFeeProcessing` to reflect the fact that only fee changes are expected here.

Reply:
I intentionally made it more generic for flexibility, but I didn't have any specific use cases in mind. I'm fine with `postTxApplyFeeProcessing`.

Reply:
I just think that if we want something more generic, we shouldn't go with raw ledger entry changes, as changes would become hard to interpret.

Reply:
Yeah that's valid. We can extend `TransactionResultMetaV1` if we need to.